util/godfig/src/lib.rs

```diff
@@ -1,3 +1,7 @@
+pub mod backend;
+pub mod godfig;
+pub use godfig::*;
```

Comment:
Perhaps make `godfig` a private mod if everything in it can be imported from the crate root.